config/cable.yml

```diff
@@ -1,9 +1,9 @@
-development:
-  adapter: async
+# development:
+#   adapter: async
 
-test:
-  adapter: async
+# test:
+#   adapter: async
 
-production:
-  adapter: redis
-  url: redis://localhost:6379/1
+# production:
+#   adapter: redis
+#   url: redis://localhost:6379/1
```

Comment:
The upgrade task suggested we add some more config in here; instead I commented it all out since it looked like we were using this but we're not.